packages/calcite-components/src/components/combobox-item-group/combobox-item-group.scss

```diff
@@ -45,8 +39,7 @@
     flex-1
     border-b
     font-bold;
-  padding-block: var(--calcite-combobox-item-spacing-unit-l);
-  padding-inline: var(--calcite-combobox-item-spacing-unit-s);
+  padding: var(--calcite-combobox-item-spacing-indent);
```

Comment:
nitpick: maybe `indent` doesn't make sense in the var name anymore? Since it covers both inline and block?